src/_pytest/nodes.py

```diff
@@ -214,6 +214,9 @@ def __init__(
         # Deprecated alias. Was never public. Can be removed in a few releases.
         self._store = self.stash
 
+        #: A list of exceptions that happened during teardown
+        self.teardown_exceptions: list[BaseException] = []
```

Comment:
Does this need to be a list and not an exception group?

Reply:
Ah, I didn't realize they were immutable..

Reply:
Guys, where to put tests? Can u suggest appropriate places? Looks like: testing/test_collection.py and teardown?

Reply:
https://github.com/Myp3a/pytest/pull/1

Reply:
Merged, thank you. Now we have a test to check if the fixture fails and exception is added to the list